index.html

```diff
@@ -8,6 +8,13 @@
 </head>
 <body>
     
+    <header>
+        <h1>BEST PAGE OF ALL TIME 🥳</h1>
+
+        <div class="nav">
+            <button>About Page</button>
+        </div>
```

Comment:
`<nav>` is probably the right element to be using here, because it's more semantic.
```suggestion
        <nav>
            <button>About Page</button>
        </nav>
```